src/CodeGeneration.Roslyn.BuildTime/CodeGeneration.Roslyn.BuildTime.csproj

```diff
@@ -1,19 +1,18 @@
-﻿<Project Sdk="MSBuild.Sdk.Extras">
+﻿<Project Sdk="Microsoft.Build.NoTargets">
```

Comment:
Cool! I didn't know about this. But if this project has no outputs, why do we need it? Are we packing with it even though it doesn't produce an assembly? If so, "NoTargets" sounds like a very misleading name.

Reply:
Well, yeah - it clears the "Build" target AFAIK, but Pack still works, so I used that :) So, no assembly, no compilation (considerable time saved), but package is still produced.

The name is a bit misleading indeed, I didn't notice really because I've been mostly reading the actual docs, not names 😅